src/main/java/au/com/sealink/quicktravel/client/models/PassengerType.java

```diff
@@ -0,0 +1,141 @@
+package au.com.sealink.quicktravel.client.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class PassengerType {
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("minimum_age")
+    @Expose
+    private Object minimumAge;
+    @SerializedName("maximum_age")
+    @Expose
+    private Object maximumAge;
+    @SerializedName("age_range_required")
+    @Expose
+    private Boolean ageRangeRequired;
+    @SerializedName("position")
+    @Expose
+    private Integer position;
+    @SerializedName("default_age")
+    @Expose
+    private Integer defaultAge;
+    @SerializedName("created_at")
+    @Expose
+    private Date createdAt;
```

Comment:
isn't createdAt/updatedAt a DateTime, or do we omit this detail for mobileApp (I think we should do this at the presentation layer and have a created_date/updated_date method instead.

Reply:
Date object in Java also handles time.

Reply:
Any to_string conversions are handled in the mobile application